_pages/publications.md

```diff
@@ -1,330 +1,112 @@
 
-* [**Evaluating Private vs. Shared Last-Level Caches for Energy Efficiency in Asymmetric Multi-Cores**](https://ieeexplore.ieee.org/document/6893211). A. Gutierrez, R.G. Dreslinski, and Trevor Mudge. In Proceedings of the 14th International Conference on Embedded Computer Systems: Architectures, Modeling, and Simulation (SAMOS), 2014. doi:[10.1109/SAMOS.2014.6893211](https://dx.doi.org/10.1109/SAMOS.2014.6893211).
-* [**Security Effectiveness and a Hardware Firewall for MPSoCs**](https://ieeexplore.ieee.org/document/7056871). M. D. Grammatikakis, K. Papadimitriou, P. Petrakis, A. Papagrigoriou, G. Kornaros, I. Christoforakis and M. Coppola. In 16th IEEE International Conference on High Performance Computing and Communications - Workshop on Multicore and Multithreaded Architectures and Algorithms, 2014, pp. 1032-1039 Aug. 2014. doi:[10.1109/HPCC.2014.173](https://dx.doi.org/10.1109/HPCC.2014.173).
-* [**Integrated 3D-Stacked Server Designs for Increasing Physical Density of Key-Value Stores**](https://dl.acm.org/doi/10.1145/2541940.2541951). Anthony Gutierrez, Michael Cieslak, Bharan Giridhar, Ronald G. Dreslinski, Luis Ceze, and Trevor Mudge. ASPLOS XIX. doi:[10.1145/2541940.2541951](https://dx.doi.org/10.1145/2541940.2541951).
-* [**Power / Capacity Scaling: Energy Savings With Simple Fault-Tolerant Caches**](https://dl.acm.org/doi/10.1145/2593069.2593184). Mark Gottscho, Abbas BanaiyanMofrad, Nikil Dutt, Alex Nicolau, and Puneet Gupta. DAC, 2014. doi:[10.1145/2593069.2593184](https://dx.doi.org/10.1145/2593069.2593184).
-* [**Write-Aware Replacement Policies for PCM-Based Systems**](https://academic.oup.com/comjnl/article/58/9/2000/341364). R. Rodríguez-Rodríguez, F. Castro, D. Chaver, R. Gonzalez-Alberquilla, L. Piñuel and F. Tirado. The Computer Journal, 2014. doi:[10.1093/comjnl/bxu104](https://dx.doi.org/10.1093/comjnl/bxu104).
-* [**Micro-architectural simulation of in-order and out-of-order ARM microprocessors with gem5**](https://ieeexplore.ieee.org/document/6893220). Fernando A. Endo, Damien Couroussé, Henri-Pierre Charles. 2014 International Conference on Embedded Computer Systems: Architectures, Modeling, and Simulation (SAMOS XIV). July 2014. doi:[10.1109/SAMOS.2014.6893220](https://dx.doi.org/10.1109/SAMOS.2014.6893220).
+      * **Analyzing Local RISC-V Interrupt Latencies with Virtual Prototyping**. Robert Hauser, Lukas Steffen, Florian Grützmacher, Christian Haubelt.
+      In Workshop Methoden und Beschreibungssprachen zur Modellierung und Verifikation von Schaltungen und Systemen (MBMV24), pp. 1-7, Kaiserslautern, Deutschland, Februar 2024 (to appear)
 
-## 2013<span class="anchor" data-clipboard-text="http://www.gem5.org/publications/#2013"></span>
+      You can also access this list from the `citations.bib` file located in the `m5out` folder, which is generated during the build process of gem5.
```

Comment:
Can you bring this up to the top? One of the cool things about this file, is that it's customized for each simulation. Only the first two papers (the original gem5 paper by Binkert et al. and the gem5-20 paper) need to be included as "required." Other papers will show up in the citations.bib